spec/models/hydrus/processable_spec.rb

```diff
@@ -19,8 +19,6 @@
                                                   workflow: 'hydrusAssemblyWF',
                                                   process: step,
                                                   status: 'completed')
-      allow(@go).to receive_message_chain(:workflows, :workflow_step_is_done).and_return(false)
-      expect(@go).to receive(:workflows_content_is_stale)
```

Comment:
As mentioned above, these methods are no longer needed since workflow stuff is handled by an external service with DS8.